support/user-guide/subaccounts.md

```diff
@@ -211,4 +217,18 @@ Sending domain and tracking domain functionality afforded to subaccount users ar
 
 **Note**: When making API requests for the above reports, the subaccount does not need to specify the subaccount ID in their payloads. Even if this is included, no matter the value, it will always be overwritten since the subaccount ID is tied to the API key associated with it. Due to this restriction, a subaccount will not be able to spoof a different subaccount ID and get access to any other subaccount's or master account's data or assets.
 
+### Subaccount UI Access
+*Limited access. Reach out to our support team or your Technical Account Manager to have this option enabled for your account.
+
+Users with the "Reporting" role can be assigned to a specific subaccount. These users only have access to information about that subaccount within the SparkPost UI. They can access their email analytics, including SparkPost Signals, as well as their events. They also have read-only access to templates and snippets associated with their specific subaccount or shared with all subaccounts.
+
+Other details to keep in mind:
+* Subaccount Reporting users can see their subaccount name and ID in the app.
+* A Reporting user can only be assigned to a subaccount at account creation. If you want to assign a current user to a specific subaccount, you will need to delete and re-add the user.
+* A Reporting user cannot be switched to a different subaccount or be changed to a the master account. If you want to make these changes, you will need to delete and re-add the user.
+* A Reporting user can only be assigned to a single subaccount. Multi-subaccount access is not supported at this time.
```

Comment:
I think this line was only appropriate in early beta docs, not public docs.

Reply:
remove the last sentence